lua/telescope/builtin/internal.lua

```diff
@@ -297,6 +297,36 @@ internal.command_history = function(opts)
   }):find()
 end
 
+internal.search_history = function(opts)
```

Comment:
There are some work going on to add inline function documentation. It would be awesome if you can add an inline documentation for the search picker in internal.lua and for the new action added.

Reply:
@tami5 I am happy to do so. Do you have an example of such docs somewhere already?